causalimpact/plot.py

```diff
@@ -36,7 +37,8 @@ def plot(
         Indicates which plot should be considered in the graphics.
       figsize: tuple.
         Changes the size of the graphics plotted.
-
+      show: bool.
+        If true, runs plt.show(), i.e., displays the figure. Defaults to True.
```

Comment:
We could also explain why this is used, that is, when `show=False` the client can have access to the axis.